### DISCUSSION TITLE
电信宽带无法拨号

### DISCUSSION THREAD

**Discussion by Futurescope** [coolsnowwolf/lede#9561] [Bugs / 问题反馈]
[拨号提示尝试连接失败、未知错误（USER_REQUEST）、未知错误（NEGOTIATION_FAILED）三者之一。只把istore固件的wan口由原先的静态改成pppoe拨号就会提示这个错误。尝试过换eth口+改wan口也不好用。电脑以及小米硬路由可以拨号。
附日志
`Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' is now down
Mon Jun 13 12:36:43 2022 daemon.info avahi-daemon[3759]: Interface eth0.IPv6 no longer relevant for mDNS.
Mon Jun 13 12:36:43 2022 daemon.info avahi-daemon[3759]: Leaving mDNS multicast group on interface eth0.IPv6 with address fe80::24e8:4dff:fe08:5e50.
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' is disabled
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' has link connectivity loss
Mon Jun 13 12:36:43 2022 daemon.info avahi-daemon[3759]: Withdrawing address record for fe80::24e8:4dff:fe08:5e50 on eth0.
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.384054] Generic PHY stmmac-0:00: attached PHY driver [Generic PHY] (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.395576] dwmac4: Master AXI performs any burst length
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.395615] rk_gmac-dwmac fe2a0000.ethernet eth0: No Safety Features support found
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.395635] rk_gmac-dwmac fe2a0000.ethernet eth0: IEEE 1588-2008 Advanced Timestamp supported
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.395778] rk_gmac-dwmac fe2a0000.ethernet eth0: registered PTP clock
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.396144] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' is enabled
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' has link connectivity
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' is setting up now
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Network device 'eth0' link is down
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' has link connectivity loss
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' is now down
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' is disabled
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.438189] Generic PHY stmmac-0:00: attached PHY driver [Generic PHY] (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.459291] dwmac4: Master AXI performs any burst length
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.459335] rk_gmac-dwmac fe2a0000.ethernet eth0: No Safety Features support found
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.459355] rk_gmac-dwmac fe2a0000.ethernet eth0: IEEE 1588-2008 Advanced Timestamp supported
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.459497] rk_gmac-dwmac fe2a0000.ethernet eth0: registered PTP clock
Mon Jun 13 12:36:43 2022 daemon.notice netifd: Interface 'wan' is enabled
Mon Jun 13 12:36:43 2022 kern.info kernel: [  249.459967] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
Mon Jun 13 12:36:47 2022 daemon.notice netifd: Network device 'eth0' link is up
Mon Jun 13 12:36:47 2022 daemon.notice netifd: Interface 'wan' has link connectivity
Mon Jun 13 12:36:47 2022 daemon.notice netifd: Interface 'wan' is setting up now
Mon Jun 13 12:36:47 2022 kern.info kernel: [  253.520490] rk_gmac-dwmac fe2a0000.ethernet eth0: Link is Up - 1Gbps/Full - flow control off
Mon Jun 13 12:36:47 2022 kern.info kernel: [  253.520541] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
Mon Jun 13 12:36:47 2022 daemon.err insmod: module is already loaded - slhc
Mon Jun 13 12:36:47 2022 daemon.err insmod: module is already loaded - ppp_generic
Mon Jun 13 12:36:47 2022 daemon.err insmod: module is already loaded - pppox
Mon Jun 13 12:36:47 2022 daemon.err insmod: module is already loaded - pppoe
Mon Jun 13 12:36:47 2022 daemon.info pppd[10106]: Plugin rp-pppoe.so loaded.
Mon Jun 13 12:36:47 2022 daemon.info pppd[10106]: RP-PPPoE plugin version 3.8p compiled against pppd 2.4.8
Mon Jun 13 12:36:47 2022 daemon.notice pppd[10106]: pppd 2.4.8 started by root, uid 0
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]: Send PPPOE Discovery V1T1 PADI session 0x0 length 12
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]:  dst ff:ff:ff:ff:ff:ff  src 26:e8:4d:08:5e:50
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]:  [service-name] [host-uniq  7a 27 00 00]
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]: Recv PPPOE Discovery V1T1 PADO session 0x0 length 37
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]:  dst 26:e8:4d:08:5e:50  src 28:6e:17:62:53:20
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]:  [service-name] [host-uniq  7a 27 00 00] [AC-name HA-LY-JXTJL-BAS-1.MAN]
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]: Send PPPOE Discovery V1T1 PADR session 0x0 length 12
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]:  dst 28:6e:17:62:53:20  src 26:e8:4d:08:5e:50
Mon Jun 13 12:36:47 2022 daemon.debug pppd[10106]:  [service-name] [host-uniq  7a 27 00 00]
Mon Jun 13 12:36:49 2022 daemon.info avahi-daemon[3759]: Joining mDNS multicast group on interface eth0.IPv6 with address fe80::24e8:4dff:fe08:5e50.
Mon Jun 13 12:36:49 2022 daemon.info avahi-daemon[3759]: New relevant interface eth0.IPv6 for mDNS.
Mon Jun 13 12:36:49 2022 daemon.info avahi-daemon[3759]: Registering new address record for fe80::24e8:4dff:fe08:5e50 on eth0.*.
Mon Jun 13 12:36:52 2022 daemon.debug pppd[10106]: Send PPPOE Discovery V1T1 PADR session 0x0 length 12
Mon Jun 13 12:36:52 2022 daemon.debug pppd[10106]:  dst 28:6e:17:62:53:20  src 26:e8:4d:08:5e:50
Mon Jun 13 12:36:52 2022 daemon.debug pppd[10106]:  [service-name] [host-uniq  7a 27 00 00]
Mon Jun 13 12:36:57 2022 daemon.debug pppd[10106]: Send PPPOE Discovery V1T1 PADR session 0x0 length 12
Mon Jun 13 12:36:57 2022 daemon.debug pppd[10106]:  dst 28:6e:17:62:53:20  src 26:e8:4d:08:5e:50
Mon Jun 13 12:36:57 2022 daemon.debug pppd[10106]:  [service-name] [host-uniq  7a 27 00 00]
Mon Jun 13 12:36:59 2022 daemon.debug pppd[10106]: Recv PPPOE Discovery V1T1 PADT session 0x6cf3 length 0
Mon Jun 13 12:36:59 2022 daemon.debug pppd[10106]:  dst 26:e8:4d:08:5e:50  src 28:6e:17:62:53:20
Mon Jun 13 12:36:59 2022 daemon.debug pppd[10106]:
Mon Jun 13 12:37:02 2022 daemon.warn pppd[10106]: Timeout waiting for PADS packets
Mon Jun 13 12:37:03 2022 daemon.err pppd[10106]: Unable to complete PPPoE Discovery
Mon Jun 13 12:37:03 2022 daemon.info pppd[10106]: Exit.
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' is now down
Mon Jun 13 12:37:03 2022 daemon.info avahi-daemon[3759]: Interface eth0.IPv6 no longer relevant for mDNS.
Mon Jun 13 12:37:03 2022 daemon.info avahi-daemon[3759]: Leaving mDNS multicast group on interface eth0.IPv6 with address fe80::24e8:4dff:fe08:5e50.
Mon Jun 13 12:37:03 2022 daemon.info avahi-daemon[3759]: Withdrawing address record for fe80::24e8:4dff:fe08:5e50 on eth0.
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' is disabled
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.674235] Generic PHY stmmac-0:00: attached PHY driver [Generic PHY] (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' is enabled
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' is setting up now
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.695473] dwmac4: Master AXI performs any burst length
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.695508] rk_gmac-dwmac fe2a0000.ethernet eth0: No Safety Features support found
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.695527] rk_gmac-dwmac fe2a0000.ethernet eth0: IEEE 1588-2008 Advanced Timestamp supported
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.695657] rk_gmac-dwmac fe2a0000.ethernet eth0: registered PTP clock
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.695985] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Network device 'eth0' link is down
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' has link connectivity loss
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' is now down
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' is disabled
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.732764] Generic PHY stmmac-0:00: attached PHY driver [Generic PHY] (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
Mon Jun 13 12:37:03 2022 daemon.notice netifd: Interface 'wan' is enabled
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.754088] dwmac4: Master AXI performs any burst length
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.754141] rk_gmac-dwmac fe2a0000.ethernet eth0: No Safety Features support found
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.754164] rk_gmac-dwmac fe2a0000.ethernet eth0: IEEE 1588-2008 Advanced Timestamp supported
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.754442] rk_gmac-dwmac fe2a0000.ethernet eth0: registered PTP clock
Mon Jun 13 12:37:03 2022 kern.info kernel: [  268.754929] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
Mon Jun 13 12:37:07 2022 daemon.notice netifd: Network device 'eth0' link is up
Mon Jun 13 12:37:07 2022 daemon.notice netifd: Interface 'wan' has link connectivity
Mon Jun 13 12:37:07 2022 daemon.notice netifd: Interface 'wan' is setting up now
Mon Jun 13 12:37:07 2022 kern.info kernel: [  272.800532] rk_gmac-dwmac fe2a0000.ethernet eth0: Link is Up - 1Gbps/Full - flow control off
Mon Jun 13 12:37:07 2022 kern.info kernel: [  272.800584] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
Mon Jun 13 12:37:07 2022 daemon.err insmod: module is already loaded - slhc
Mon Jun 13 12:37:07 2022 daemon.err insmod: module is already loaded - ppp_generic
Mon Jun 13 12:37:07 2022 daemon.err insmod: module is already loaded - pppox
Mon Jun 13 12:37:07 2022 daemon.err insmod: module is already loaded - pppoe
Mon Jun 13 12:37:07 2022 daemon.info pppd[10270]: Plugin rp-pppoe.so loaded.
Mon Jun 13 12:37:07 2022 daemon.info pppd[10270]: RP-PPPoE plugin version 3.8p compiled against pppd 2.4.8
Mon Jun 13 12:37:07 2022 daemon.notice pppd[10270]: pppd 2.4.8 started by root, uid 0
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]: Send PPPOE Discovery V1T1 PADI session 0x0 length 12
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  dst ff:ff:ff:ff:ff:ff  src 26:e8:4d:08:5e:50
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  [service-name] [host-uniq  1e 28 00 00]
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]: Recv PPPOE Discovery V1T1 PADO session 0x0 length 37
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  dst 26:e8:4d:08:5e:50  src 28:6e:17:62:53:20
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  [service-name] [host-uniq  1e 28 00 00] [AC-name HA-LY-JXTJL-BAS-1.MAN]
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]: Send PPPOE Discovery V1T1 PADR session 0x0 length 12
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  dst 28:6e:17:62:53:20  src 26:e8:4d:08:5e:50
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  [service-name] [host-uniq  1e 28 00 00]
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]: Recv PPPOE Discovery V1T1 PADS session 0x6111 length 12
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  dst 26:e8:4d:08:5e:50  src 28:6e:17:62:53:20
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]:  [service-name] [host-uniq  1e 28 00 00]
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]: PADS: Service-Name: ''
Mon Jun 13 12:37:07 2022 daemon.info pppd[10270]: PPP session is 24849
Mon Jun 13 12:37:07 2022 daemon.warn pppd[10270]: Connected to 28:6e:17:62:53:20 via interface eth0
Mon Jun 1
![屏幕截图 2022-06-13 120525](https://user-images.githubusercontent.com/34177138/173281649-fc967faf-59b5-446d-8692-73e5b4a3e7b8.png)
![屏幕截图 2022-06-13 123852](https://user-images.githubusercontent.com/34177138/173281654-c1c87fbb-ba7f-48dc-87e7-0b0ef9ea1afe.png)
3 12:37:07 2022 daemon.debug pppd[10270]: using channel 5
Mon Jun 13 12:37:07 2022 kern.info kernel: [  272.921224] pppoe-wan: renamed from ppp0
Mon Jun 13 12:37:07 2022 daemon.info pppd[10270]: Renamed interface ppp0 to pppoe-wan
Mon Jun 13 12:37:07 2022 daemon.info pppd[10270]: Using interface pppoe-wan
Mon Jun 13 12:37:07 2022 daemon.notice pppd[10270]: Connect: pppoe-wan <--> eth0
Mon Jun 13 12:37:07 2022 daemon.debug pppd[10270]: sent [LCP ConfReq id=0x1 <mru 1492> <magic 0x82556ff>]
Mon Jun 13 12:37:07 2022 user.notice net-smp-affinity: enable eth2 hw offload SG and TSO
Mon Jun 13 12:37:07 2022 user.notice net-smp-affinity: enable eth3 hw offload SG and TSO
Mon Jun 13 12:37:08 2022 daemon.info avahi-daemon[3759]: Joining mDNS multicast group on interface eth0.IPv6 with address fe80::24e8:4dff:fe08:5e50.
Mon Jun 13 12:37:08 2022 daemon.info avahi-daemon[3759]: New relevant interface eth0.IPv6 for mDNS.
Mon Jun 13 12:37:08 2022 daemon.info avahi-daemon[3759]: Registering new address record for fe80::24e8:4dff:fe08:5e50 on eth0.*.
Mon Jun 13 12:37:10 2022 daemon.debug pppd[10270]: sent [LCP ConfReq id=0x1 <mru 1492> <magic 0x82556ff>]
Mon Jun 13 12:37:13 2022 daemon.debug pppd[10270]: sent [LCP ConfReq id=0x1 <mru 1492> <magic 0x82556ff>]
Mon Jun 13 12:37:16 2022 daemon.debug pppd[10270]: sent [LCP ConfReq id=0x1 <mru 1492> <magic 0x82556ff>]
Mon Jun 13 12:37:19 2022 daemon.debug pppd[10270]: sent [LCP ConfReq id=0x1 <mru 1492> <magic 0x82556ff>]
Mon Jun 13 12:37:22 2022 daemon.debug pppd[10270]: sent [LCP ConfReq id=0x1 <mru 1492> <magic 0x82556ff>]`

**Comment by Futurescope** [comment]
有点乱，日志跟截图贴在这边
![屏幕截图 2022-06-13 120525](https://user-images.githubusercontent.com/34177138/173281794-8b157bc5-ee25-4cd0-9b80-ba99324e5bd2.png)
![屏幕截图 2022-06-13 123852](https://user-images.githubusercontent.com/34177138/173281803-82196866-4122-4723-9645-0717d817a4b7.png)
[ppp_log.txt](https://github.com/coolsnowwolf/lede/files/8887960/ppp_log.txt)

**Comment by xiexiaosheng** [comment]
istore固件的问题反馈，好像不在这边

**Reply by Futurescope** [reply]
国际版0611固件也有这个问题，相比istore来说没有未知错误的提示，但也是拿不到wan口ip
